### ISSUE TITLE
Mallory-Denk inclusion bodies

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10067]
**Preferred term label:**
Mallory-Denk inclusion bodies

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/1787460

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cryptogenic cirrhosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000457
- label: Mallory bodies
- parents: [Abnormal hepatocyte morphology (HP:0034713)](https://hpo.jax.org/app/browse/term/HP:0034713)
- pmid: PMID:31424884
- def: Cytoplasmic hyaline inclusions of hepatocytes that are predominantly filamentous ranging from a diameter of 3 to 24 nm vs. 10 nm of classical IF. Mallory-Denk bodies can be classified as type I (parallels filaments), II (randomly orient filaments), or III (granular and amorphous). Type II occurs in the periphery while type III occurs around the center. MDB occur in ballooned hepatocytes.
- comment: Mallory bodies were once thought to be specific for alcoholic hepatitis but now are known to occur in other liver diseases which include nonalcoholic steatohepatitis, cholestatic liver diseases, primary biliary cirrhosis and hepatocellular carcinoma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10067
- synonyms: Mallory-Denk bodies(NONE)
